.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
JSON does not support comments. I am surprised if this is working.

Reply:
Yeah @algomaster99, JSON won't support comments but since it's in `.vscode/` dir it works well

Reply:
> Yeah @algomaster99, JSON won't support comments but since it's in .vscode/ dir it works well

Could you please cite some reference?

Reply:
Sure!
Have a look at this :
https://code.visualstudio.com/docs/languages/json#_json-with-comments

Reply:
Technically, this is not JSON. You won't be able to use a standard JSON parser without stripping comments first. But you can use a simple, JSON-like language with comments for config.

Reply:
Thanks, @Aniruddha-Shriwant !